site/app/views/NavigationView.php

```diff
@@ -5,6 +5,14 @@
 use \app\libraries\GradeableType;
 use app\models\Gradeable;
 
+$default_zone = "America/New_York";
+$timezone = file_get_contents("/etc/timezone");
+if (!strlen($timezone)) {
+    $timezone = $default_zone;
+}
+$timezone = trim($timezone);
+date_default_timezone_set($timezone);
+
```

Comment:
Unnecessary as we're already setting the `date_default_timezone_set` within `site/public/index.php`.

You also shouldn't introduce 'side-effects' when loading a file that has a class in it per [PSR-1](https://www.php-fig.org/psr/psr-1/#23-side-effects).